src/utils/AccessibleButton.ts

```diff
@@ -13,7 +13,7 @@ export enum ArrowDirection {
 
 export class AccessibleButton {
   private element: Dom;
-  private hasLabel = true;
+  private hasLabelOrTitle = true;
```

Comment:
Shouldn't the default value be false? If I run this code only `new AccessibleButton()` , `hasLabelOrTitle` will be true, but `title` and `label` won't be initialized. 

If we set the default value to false, we could set it to true when we assign a value `title` or `label`  .

Reply:
Good observation! We want to ensure that every accessible button has a title or label default, that why this value is initialized as true, otherwise it will log an error to the dev.

A dev has to call the withoutLabelOrTitle() method to remove that rule.